book/integration-service.md

```diff
@@ -0,0 +1,271 @@
+# Integration Service
+
+
+## Overview
+
+The Integration Service is composed of a couple controllers that facilitate automated testing of content produced by the HACBS build pipelines. It is responsible for integration testing capabilities.
+
+To support this capability, it needs to do a couple of things.
```

Comment:
You use `it` many times in the bullets. Can we change this one to the integration service?

Reply:
Are we getting into too much detail for an overview? Would it make sense to simplify this and further break out the details for the various actions elsewhere?

Reply:
Modified overview section by adding additional sections.